public/docs/_examples/forms/dart/lib/hero_form_component.dart

```diff
@@ -0,0 +1,27 @@
+library hero_form.hero_form_component;
+
+import 'package:angular2/angular2.dart';
+import 'package:hero_form/hero.dart';
+
+const List<String> _powers = const [
+  'Really Smart',
+  'Super Flexible',
+  'Super Hot',
+  'Weather Changer'
+];
+
+@Component(selector: 'hero-form')
+@View(
+    templateUrl: '../web/hero_form_component.html',
```

Comment:
Yeah, no. Lib should NEVER reference files outside lib. `hero_form_component.html` can go in the lib dir, though – at least it should